fixtures/public/react-loader.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Take a version from the window query string and load a specific
+ * version of React.
+ *
+ * @example
+ * http://localhost:3000?version=15.4.1
+ * (Loads React 15.4.1)
+ */
+
+var REACT_PATH = 'react.js';
+var DOM_PATH = 'react-dom.js';
```

Comment:
Question got hidden as "outdated", but its not: https://github.com/facebook/react/pull/8589#discussion_r94507090